src/components/Trade/TradeInput.tsx

```diff
@@ -270,7 +270,6 @@ export const TradeInput = ({ history }: RouterProps) => {
                 alignItems='center'
                 color='gray.500'
                 fontSize='sm'
-                spacing='24px'
```

Comment:
Nice. According to the docs, that wasn't supported as a styleProp to start with, only with `<SimpleGrid />`.